e2e/pages/adf/dialog/contentList.js

```diff
@@ -28,7 +28,7 @@ var ContentList = function () {
     var actionMenu = element(by.css("div[role='menu']"));
     var optionButton = by.css("button[data-automation-id*='action_menu_']");
     var rowByRowName = by.xpath("ancestor::div[contains(@class, 'adf-datatable-row')]");
-    var nameColumn = by.css("div[id*='document-list-container'] div[class*='adf-datatable-row'] div[class*='--text full-width'] span");
+    var nameColumn = by.css('div[class*="adf-data-table-cell"][title="Display name"]');
```

Comment:
This locator will retrieve the name of the column also('Display name' in our case) that may make other tests fail. I suggest to change the locator to this one: by.css('div[class*="datatable-body"] div[class*="adf-data-table-cell"][title="Display name"]')